util/measures/additional-stratifications.json

```diff
@@ -0,0 +1,39 @@
+{
+  "CMS137v5": {
+    "0BBF8596-4CFE-47F4-A0D7-9BEAB94BA4CD": [
+      "EFB5B088-CE10-43DE-ACCD-9913B7AC12A2",
+      "94B9555F-8700-45EF-B69F-433EBEDE8051"
+    ],
+    "7FFA49C4-D708-491E-85FE-6855F0A725DF": [
+      "ABC5631A-81C0-45C9-9306-716EAE39CDDB",
+      "2654804B-E6DA-4401-AA8B-1FEEACC0C259"
+    ]
+  },
+  "CMS153v5": {
```

Comment:
are you formatting it this way to be consistent with something elsewhere? if not, consider naming the values for readability, e.g.

```
[{
  "eMeasureId": "CMS137v5",
  "something": [{
    "somethingUuid": "0BBF8596-4CFE-47F4-A0D7-9BEAB94BA4CD",
    "somethingElseUuids": ["EFB5B088-CE10-43DE-ACCD-9913B7AC12A2", "94B9555F-8700-45EF-B69F-433EBEDE8051"
  }, {
    "somethingUuid": "0BBF8596-4CFE-47F4-A0D7-9BEAB94BA4CD",
    "somethingElseUuids": ["EFB5B088-CE10-43DE-ACCD-9913B7AC12A2", "94B9555F-8700-45EF-B69F-433EBEDE8051"
  }]
}, {
  "eMeasureId": "CMS153v5",
  ...
}]
```

if this is too much effort to change at this point, maybe include a comment block above the .json filename constant in get-stratifications.js that outlines what the expected format is (since you can't put comments into .json files)

Reply:
i agree here - it'll be much easier to maintain if this is structured in a self-documenting way